src/CreditCard/checkIIN.js

```diff
@@ -11,7 +11,7 @@ const checkIIN = (cardNumber = "") =>
   )[0] || ["Your Bank"])[0];
 
 const preCheckIIN = (cardNumber = "") => {
-  const digits = +cardNumber.slice(0, 2);
+  const digits =+ cardNumber.slice(0, 2);
```

Comment:
En este caso el `+` era para "convertir" en numero, podria cambiarse a 
```suggestion
  const digits = Number(cardNumber.slice(0, 2)) || 0;
```

Para que sea mas claro